arduino-ide-extension/src/browser/serial/monitor/monitor-view-contribution.tsx

```diff
@@ -52,6 +52,14 @@ export namespace SerialMonitor {
       },
       'vscode/output.contribution/clearOutput.label'
     );
+    export const COPY_OUTPUT = Command.toLocalizedCommand(
+      {
+        id: 'serial-monitor-copy-output',
+        label: 'Copy Output',
+        iconClass: codicon('copy'),
+      },
+      'arduino/serial/copyOutput'
+    );
```

Comment:
```suggestion
    export const COPY_OUTPUT = {
      id: 'serial-monitor-copy-output',
    };
```

Attaching a `label` and `iconClass` to this command at this level is unnecessary. A `label` and `iconClass` are required for a command if the IDE wants to show it in the _Command Palette_, but it's not the case here; it will be shown on the toolbar or the _Monitor_ widget. Registering the command with the `id`, and doing the `label` and `icon` classes for the toolbar item is sufficient. See the corresponding proposed change below 👇

Reply:
Resolved in [5d47076](https://github.com/arduino/arduino-ide/pull/2718/commits/5d470766178afdcf9103fdf3c35639086fc63fa8)